waterlog/src/reducers/SegmentLeaksReducer.js

```diff
@@ -1,27 +1,36 @@
 import {
-    FETCH_SEGMENTS_LEAKS
+    FETCH_SEGMENTS_LEAKS_BEGIN, FETCH_SEGMENTS_LEAKS_SUCCESS, FETCH_SEGMENTS_LEAKS_FAILURE
```

Comment:
This is a suggestion not request: 
each import on its own line

Reply:
Resolve after read I do not mind

Reply:
I will take into consideration